moderntribe/square1-global-docker
# Feature/installer cleanup

- Combines the installer/dev installer into a single installer.
 - README improvements.

**Test dev install**

1. Clone this repo
2. Check out this branch
3. Run `bash -c "(export SO_DEV=true; ./install/install.sh)"` from the root of the project.
